tests/tests_deployment/test_jupyterhub_api.py

```diff
@@ -52,6 +56,44 @@ def test_check_default_roles_added_in_keycloak():
     role_names = [role["name"] for role in client_roles]
     assert "allow-app-sharing-role" in role_names
     assert "allow-read-access-to-services-role" in role_names
+    assert "allow-group-directory-creation-role" in role_names
+
+
+@pytest.mark.parametrize(
+    "component,scopes",
+    (["shared-directory", "write:shared-mount"],),
+)
+@pytest.mark.filterwarnings(
+    "ignore:.*auto_refresh_token is deprecated:DeprecationWarning"
+)
+@pytest.mark.filterwarnings("ignore::urllib3.exceptions.InsecureRequestWarning")
+def test_check_directory_creation_scope_attributes(component, scopes):
+    client_role = get_keycloak_client_role(
+        client_name="jupyterhub", role_name="allow-group-directory-creation-role"
+    )
+    assert client_role["attributes"]["component"][0] == component
+    assert client_role["attributes"]["scopes"][0] == scopes
+
+
+@pytest.mark.parametrize(
+    "groups_to_volume_mount",
+    (["/developer", "/admin", "/analyst"],),
```

Comment:
Since, this parametrize is only used for a single condition, it's probably not required, worth just using it explicitly in the function. (same for the previous function `test_check_directory_creation_scope_attributes`)

Reply:
done